instagram/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import dummyData from './dummy-data.js'
+import { PostContainer } from './components/PostContainer/PostContainer.js'
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+       DummyData: [],
+     }
+  };
+  
+  componentDidMount(){
+    this.setState({DummyData: dummyData});
```

Comment:
If you rename `this.state.DummyData` to `this.state.dummyData`, you can use the ES6 object shorthand here

```this.setState({ dummyData });```